CompanyManager.DAL/Entities/Person.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompanyManager.DAL.Entities
+{
+    public class Person
+    {
+        public int Id { get; set; }
+
+        public int PostId { get; set; }
+
+        public string Name { get; set; }
```

Comment:
Ja osobiście wolał bym, żeby to się nazywało FirstName na przykład, bo często Name może mylić się z pełnym imieniem i nazwiskiem, a chodzi nam tylko o imię.